packages/api/src/webhooks/cannon.ts

```diff
@@ -558,16 +561,44 @@ export default class WebhookCannon {
 
       const os = await db.objectStore.get(this.recordCatalystObjectStoreId);
       // we can't rate limit this task because it's not a user action
+      let url = pathJoin(
+        os.publicUrl,
+        session.playbackId,
+        session.id,
+        "output.m3u8"
+      );
+
+      const secondaryOs = await db.objectStore.get(
+        this.secondaryRecordObjectStoreId
+      );
+      if (secondaryOs) {
+        let params = {
+          method: "GET",
+          timeout: 5 * 1000,
+        };
+        const resp = await fetchWithTimeout(url, params);
```

Comment:
I _think_ `fetchWithTimeout` might throw on timeout, so we prolly want a try/catch here (might be worth a helper func like `fileExists(url)`)